web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
```

Comment:
The brief description doesn't use the imperative, so it would be "Charts". Does the "X-Y" before area chart add anything? And the double "Chart … chart" is inconvenient. Maybe "Visualizes event data with an area chart." is a good start, feel free to improve.